### ISSUE TITLE
Enhance the Upload Form

### ISSUE THREAD

**Issue by laceysanderson** [UofS-Pulse-Binfo/analyzedphenotypes#21]
Positive Feedback:
- The flow and descign of the upload form is good overall
- I like the "You have fully described # of 4 Traits. Please ensure to describe all traits before clicking the Next Step button" warning at the bottom of step3
- It's nice that clicking on the suggestions, fills in the field!

The following include suggested improvements for the Upload Form:
- Make image optional but include statement highly encouraging you to upload it if applicable.
- allow edit of the unit?
- hide the entire "mapping information" fieldset if crop ontology is not set.
- "Mapping information" is ambiguous to researchers. Instead we should call this section "Related Ontology Terms"
- "Description" title should become "Description of Trait including Collection Method" with a #description of "Please fully describe the trait and the method you used to collect this data."
- "You have fully described # of 4 Traits. Please ensure to describe all traits before clicking the Next Step button" should update even if the crop ontology is not set since it's optional.

**Comment by laceysanderson** [comment]
On hold until all outstanding PRs are merged.

**Comment by laceysanderson** [comment]
Additionally, a link to the uploaded data in the bean chart would be helpful. As it is, once the upload is complete I'm left wondering: "now what? Where do I go to check it?"

**Comment by laceysanderson** [comment]
So testing this last time I ran into a UX issue. I got to the validation step, watched the progress bar and then incorrectly assumed I was done 😝 Perhaps we should only show the progress bar for the loading step (4) and the validation step (2) should be a simple spinner (maybe the awesome plant navy one we have)?

**Comment by laceysanderson** [comment]
PR Merged 👍
